test/services/prompter/database-prompts.js

```diff
@@ -0,0 +1,768 @@
+            expect(processStub.calledOnce).to.equal(true);
+            expect(processStub.calledWith(1)).to.equal(true);
+          });
+
+          it('should log an error message', async () => {
+            const message = 'Cannot parse the database dialect. Please, check the syntax of the database connection string.';
+
+            expect(loggerSpy.calledOnce).to.equal(true);
+            expect(loggerSpy.getCall(0).args[0]).to.equal(message);
+          });
+
+          it('should send an event', async () => {
+            const firstArgument = 'unknown_error';
+            const secondArgument = 'Cannot parse the database dialect. Please, check the syntax of the database connection string.';
+
+            expect(eventSenderSpy.calledOnce).to.equal(true);
+            expect(eventSenderSpy.getCall(0).args[0]).to.equal(firstArgument);
+            expect(eventSenderSpy.getCall(0).args[1]).to.equal(secondArgument);
+          });
+        });
+      });
+    });
+
+    describe('When the dbConnectionUrl option is not requested', () => {
+      const databasePrompts = new DatabasePrompts(program, envConfig, prompts, requests);
+
+      it('should not do anything', async () => {
+        expect(envConfig.dbConnectionUrl).to.equal(undefined);
+        expect(prompts).to.have.lengthOf(0);
+
+        await databasePrompts.handleConnectionUrl();
+
+        expect(envConfig.dbConnectionUrl).to.equal(undefined);
+        expect(prompts).to.have.lengthOf(0);
+      });
+    });
+  });
+
+  describe('Handling dialect : ', () => {
+    describe('When the dbDialect option is requested', () => {
+      describe('not using windows', () => {
+        let databasePrompts;
+
+        before(() => {
+          requests.push('dbDialect');
+
+          databasePrompts = new DatabasePrompts(program, envConfig, prompts, requests);
+          databasePrompts.handleDialect();
+        });
+
+        after(() => {
+          resetParams();
+        });
+
+        it('should add a prompt to ask for it', () => {
+          expect(prompts).to.have.lengthOf(1);
+        });
+
+        it('should add a prompt with the correct configuration', () => {
+          expect(prompts[0].type).to.equal('list');
+          expect(prompts[0].name).to.equal('dbDialect');
+          expect(prompts[0].message).to.equal('What\'s the database type? ');
+          expect(prompts[0].choices).to.deep.equal(['postgres', 'mysql', 'mssql', 'mongodb']);
+        });
+
+        it('should not change the configuration', () => {
+          expect(envConfig.dbDialect).to.equal(undefined);
+        });
+      });
+
+      describe('using windows', async () => {
+        let databasePrompts;
+        let platformStub;
+
+        before(() => {
+          requests.push('dbDialect');
+          databasePrompts = new DatabasePrompts(program, envConfig, prompts, requests);
+          platformStub = sinon.stub(process, 'platform').value('win');
```

Comment:
```suggestion
          platformStub = sinon.stub(process, 'platform').value('win32');
```
If I follow the documentation it should be win32 I think: https://nodejs.org/api/process.html#process_process_platform